dahyun1226/app/src/main/java/com/example/android_toy_project_study_2020_mvvm/data/GithubDetailRepoData.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.android_toy_project_study_2020_mvvm.data
+
+data class GithubDetailRepoData (
+    val owner: GithubOwnerData,
+    val full_name: String,
```

Comment:
SerializedName 어노테이션을 통해 데이터 변수와 앱에서 사용할 변수의 이름을 매칭할 수 있습니다.
보통 앱에서 사용하는 변수는 카멜 케이스를 사용하기 때문에 다른 부분도 아래와 같이 수정해주세요.

```suggestion
 @SerializedName("full_name")
  val fullName: String,
```

Reply:
바꾸었습니다. underscore 있는 것만 바꿔도 괜찮을까요?

Reply:
커밋 번호 부탁드립니다.

GithubDetailRepoData 에 있는 모든 변수 및 다른 data class도 전부 변경 해주세요.

Reply:
일괄 변경 완료
110a334